src/test/resources/transformerReactivePowerControlOuterLoopReport.txt

```diff
@@ -0,0 +1,256 @@
++ Test Report
+  + Load flow on network 'two-windings-transformer-control'
+    + Network CC0 SC0
+       AC load flow completed successfully (solverStatus=CONVERGED, outerloopStatus=STABLE)
+      + Post loading processing
```

Comment:
It took me time to understand what is a "post loading processing"... can we find a better name?

Reply:
I propose "Network info"